testing/src/main/java/org/conscrypt/java/security/StandardNames.java

```diff
@@ -357,7 +357,7 @@ private static void addOpenSsl(String cipherSuite) {
             "TLS_PSK_WITH_AES_256_CBC_SHA");
 
     // Should be updated to match BoringSSL's defaults when they change.
-    // https://boringssl.googlesource.com/boringssl/+/master/ssl/t1_lib.cc#289
+    // https://boringssl.googlesource.com/boringssl/+/main/ssl/extensions.cc#215
```

Comment:
TBH this is just going to get out of date again later, but I will deal with that another time. :-)